src/dataflow/src/render/reduce.rs

```diff
@@ -853,6 +1261,19 @@ where
         )
 }
 
+/// Transforms a vector containing indexes of needed columns into one containing
+/// the "skips" an iterator over a Row would need to perform to see those values.
+///
+/// E.g. [3, 6, 10, 15] turns into [3, 3, 4, 5]
+fn convert_indexes_to_skips(mut indexes: Vec<usize>) -> Vec<usize> {
```

Comment:
This has requirements for correctness that perhaps we should check, if we want a re-useable method. For example, these should be strictly increasing indexes, rather than any old indexes in any order.

Reply:
added a log line if things arent strictly increasing